deploy/setup_test.sh

```
@@ -0,0 +1,171 @@
+            mv "$tf" "$envfile"
+        }
+
+        if "$@"; then
+            echo "PASS: $name"
+            exit 0
+        else
+            echo "FAIL: $name"
+            exit 1
+        fi
+    )
+    if [[ $? -eq 0 ]]; then
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if mycmd;', not indirectly with $?.

(SC2181)